G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookNewMessageCommand.cs

```diff
@@ -34,6 +34,9 @@ public class Arguments : CommandArguments
 
             [Argument]
             public VariableStructure Result { get; set; } = new VariableStructure("result");
+
+            [Argument(Required = false, Tooltip = "Value indicating whether the mail message body is in HTML")]
+            public BooleanStructure IsBodyHtml { get; set; } = new BooleanStructure(false);
```

Comment:
Discuss name of this argument with Przemek.

Reply:
Przemek approved this name